This patch fixes the multilib generation for the mips64vr*-elf* targets.
It should be submitted upstream...

2009-07-16  Nick Clifton  <nickc@redhat.com>

	* config/mips/vr.h (MULTILIB_ABI_DEFAULT): Define.


Index: gcc/config/mips/vr.h
===================================================================
RCS file: /cvs/cvsfiles/gnupro/gcc/config/mips/vr.h,v
retrieving revision 1.17
diff -c -3 -p -r1.17 vr.h
*** gcc/config/mips/vr.h	18 Mar 2009 17:14:29 -0000	1.17
--- gcc/config/mips/vr.h	16 Jul 2009 16:57:24 -0000
*************** along with GCC; see the file COPYING3.  
*** 20,34 ****
  <http://www.gnu.org/licenses/>.  */
  
  #define DEFAULT_VR_ARCH "mfix-vr4130"
! #undef MIPS_ABI_DEFAULT
! #define MIPS_ABI_DEFAULT ABI_EABI
! #undef MULTILIB_DEFAULTS
! #define MULTILIB_DEFAULTS \
! 	{ MULTILIB_ENDIAN_DEFAULT,		\
! 	  MULTILIB_ABI_DEFAULT,			\
  	  DEFAULT_VR_ARCH }
  
! #undef DRIVER_SELF_SPECS
  #define DRIVER_SELF_SPECS \
  	/* Enforce the default architecture.  This is mostly for	\
  	   the assembler's benefit.  */					\
--- 20,36 ----
  <http://www.gnu.org/licenses/>.  */
  
  #define DEFAULT_VR_ARCH "mfix-vr4130"
! #undef  MIPS_ABI_DEFAULT
! #define MIPS_ABI_DEFAULT     ABI_EABI
! #undef  MULTILIB_ABI_DEFAULT
! #define MULTILIB_ABI_DEFAULT "mabi=eabi"
! #undef  MULTILIB_DEFAULTS
! #define MULTILIB_DEFAULTS 		\
! 	{ MULTILIB_ENDIAN_DEFAULT,	\
! 	  MULTILIB_ABI_DEFAULT,		\
  	  DEFAULT_VR_ARCH }
  
! #undef  DRIVER_SELF_SPECS
  #define DRIVER_SELF_SPECS \
  	/* Enforce the default architecture.  This is mostly for	\
  	   the assembler's benefit.  */					\
